src/main/java/org/folio/rest/util/HelperUtils.java

```diff
@@ -200,15 +207,25 @@ public static Void handleErrorResponse(Handler<AsyncResult<javax.ws.rs.core.Resp
 
   public static boolean isErrorMessageJson(String errorMessage) {
     if (!StringUtils.isEmpty(errorMessage)) {
-      Pattern pattern = Pattern.compile("(message).*(code).*(parameters)");
-      Matcher matcher = pattern.matcher(errorMessage);
+      Matcher matcher = ERROR_PATTERN.matcher(errorMessage);
       if (matcher.find()) {
         return matcher.groupCount() == 3;
       }
     }
     return false;
   }
 
+  public static boolean isErrorsMessageJson(String errorsMessage) {
+    if (!StringUtils.isEmpty(errorsMessage)) {
+      errorsMessage = errorsMessage.replaceAll("\r\n", "");
```

Comment:
We have sonarcube warning, Sonar propose to use replace()

Reply:
Replaced to replace()